cli/src/android/update.ts

```diff
@@ -102,7 +115,7 @@ export async function handleCordovaPluginsGradle(config: Config,  cordovaPlugins
       if (!framework.$.type && !framework.$.custom) {
         frameworksArray.push(framework.$.src);
       } else if (framework.$.custom && framework.$.custom === 'true' && framework.$.type && framework.$.type === 'gradleReference') {
-        applyArray.push(`apply from: "../../../../${p.id}/${framework.$.src}"`);
+        applyArray.push(`apply from: "${relative(pluginsFolder, p.rootPath)}/${framework.$.src}"`);
```

Comment:
here you also need to replace the `\` with `/` on the relative result.

Reply:
Rha, missed it!